pkgs/development/python-modules/pyftdi/default.nix

```diff
@@ -1,26 +1,31 @@
 { lib
 , buildPythonPackage
-, fetchPypi
+, fetchFromGitHub
 , pythonOlder
 , pyusb
 , pyserial
 }:
 
 buildPythonPackage rec {
   pname = "pyftdi";
-  version = "0.44.2";
+  version = "0.49.0";
   disabled = pythonOlder "3.5";
 
-  src = fetchPypi {
-    inherit pname version;
-    sha256 = "18k9wnpjxg71v4jm0pwr2bmksq7sckr6ylh1slf0xgpg89b27bxq";
+  src = fetchFromGitHub {
+    owner = "eblot";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "063kwvgw7g4nn09pyqwqy72vnhzw0aajg23bi32vr0k49g8fx27s";
   };
 
   propagatedBuildInputs = [ pyusb pyserial ];
 
+  doCheck = true;
```

Comment:
unfortunately, this says it executes zero tests every time, not sure what's wrong.

Reply:
from their testing docs:
```
Most unit tests are disabled, as they require specific slaves, with a dedicated HW wiring. Reproducing such test environments can be challenging, as it requires dedicated test benchs.
```
I think just importing is fine